bin/server/cmd/profile.go

```diff
@@ -18,6 +18,11 @@ func getBaseProfile() server.Profile {
 		}
 		demoDataDir = fmt.Sprintf("demo/%s", demoName)
 	}
+	if flags.backupBucket == "" {
+		flags.backupStorageBackend = api.BackupStorageBackendLocal
```

Comment:
In this case, it doesn't need to be part of flags?

Reply:
Also use
```
storageBackend := api.BackupStorageBackendLocal
if flags.backupBucket != "" {
  storageBackend = api.BackupStorageBackendS3
}
```

Reply:
Same here, I don't feel we need an extra backupStorageBackend flag.

Flags.xxx is reserved for input flag. Here we just need to capture a state in memory. And such state can be captured in backupBucket by checking its prefix.